Add sotrovimab to therapeutics (#1000)
greenelab/covid19-review: content/20.pharmaceuticals.md

The 47D11 antibody clone was able to cross-neutralize SARS-CoV-1 and SARS-CoV-2.
This antibody (now ABVV-47D11) has recently entered clinical trials in collaboration with AbbVie.
Additionally, an extensive monoclonal neutralizing antibody pipeline has been developed to combat the ongoing pandemic, with over 50 different antibodies in clinical trials [@doi:10.1016/j.bsheal.2021.02.001].

> Thus far, the monotherapy sotrovimab and two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUA emergency use authorization by the FDA.

Reviewer: ```suggestion
Thus far, the monotherapy sotrovimab and two antibody cocktails (bamlanivimab/estesevimab and casirivimab/imdevimab) have been granted EUAs by the FDA.
```

Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].

Sotrovimab is the most recent mAb to receive an EUA.

> It was identified in the memory B cells of a 2003 survivor of SARS and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y]. (look at 24, 26, 32)

Reviewer: Is `(look at 24, 26, 32)` (and also further down) an internal comment, or meant to be there?

Author: HAHA thank you for catching this!! It was my note to myself that I never cleaned up! I'll do that this morning 🤦‍♀️

This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].

> In fact, these residues are highly conserved among sarbecoviruses [@doi:10.1038/s41586-020-2349-y]. (see 39)

Reviewer: ` (see 39)` see above

Based on this data, the FDA granted an EUA for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk of developing severe disease [@url:https://www.fda.gov/news-events/press-announcements/coronavirus-covid-19-update-fda-authorizes-monoclonal-antibodies-treatment-covid-19].
Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].

Sotrovimab is the most recent mAb to receive an EUA.
It was identified in the memory B cells of a 2003 survivor of SARS [@doi:10.1038/nm1080] and was found to be cross-reactive with SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
This cross-reactivity is likely attributable to conservation within the epitope, with 17 out of 22 residues conserved between the two viruses, four conservatively substituted, and one semi-conservatively substituted [@doi:10.1038/s41586-020-2349-y].

> In fact, these residues are highly conserved among sarbecoviruses [@doi:10.1038/s41586-020-2349-y].

Reviewer 2: I would add something about how sarbecoviruses include SARS-CoV-2. Not sure this is stated elsewhere in this manuscript.

Author: ```suggestion
In fact, these residues are highly conserved among sarbecoviruses, a clade that includes SARS-CoV-1 and SARS-CoV-2 [@doi:10.1038/s41586-020-2349-y].
```

This versatility has led to it being characterized as a "super-antibody" [@doi:10.1038/s41587-021-00980-x], a potent, broadly neutralizing antibody [@doi:10.1038/nri.2017.148].
Interim analysis of data from a clinical trial [@clinicaltrials:NCT04545060] reported high safety and efficacy of this mAb in 583 COVID-19 patients [@doi:10.1101/2021.05.27.21257096].

> Sotrovimab was found to be 85% effective in reducing progression to the primary endpoint, which was the proportion of patients hospitalized for more than 24 hours or dying within 29 days of receiving sotrovimab or a placebo.

Reviewer 2: I made this suggested change directly in @rando2's add-mab branch. Not sure I was supposed to do that - sorry!

Changing this sentence to:
"Compared to placebo, sotrovimab was found to be 85% more effective in reducing progression to the primary endpoint, which was the proportion of patients who, within 29 days, were either hospitalized for more than 24 hours or died."

Author: Looks good to me!

Several strategies have been employed to try to mitigate the risk of diminished antibody neutralization.
Antibody cocktails such as those already holding an EUA may help overcome the risk for attenuation of the neutralizing activity of a single monoclonal antibody.
These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.

However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355].
Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].

> Sotrovimab, one such super-antibody, has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data/].

Reviewer: ```suggestion
Sotrovimab, one such "super-antibody", has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data/].
```

Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses [@doi:10.1126/science.abf4830].
Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.

> Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotorimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.

Reviewer: ```suggestion
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.
```

These cocktails consist of antibodies that recognize different epitopes on the spike protein, decreasing the likelihood that a single amino acid change can cause resistance to all antibodies in the cocktail.
However, neutralizing resistance can emerge even against an antibody cocktail if the individual antibodies target subdominant epitopes [@doi:10.1101/2021.02.03.429355].
Another strategy is to develop broadly neutralizing antibodies that target structures that are highly conserved, as these are less likely to mutate [@doi:10.1016/j.immuni.2007.11.018; @doi:10.1126/science.1171491] or to target epitopes that are insensitive to mutations [@doi:10.1016/j.vaccine.2006.04.054].

> Sotrovimab, one such "super-antibody", is thought to be somewhat robust to neutralization escape [@doi:10.1086/651022] and has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data/].

Reviewer 3: ```suggestion
Sotrovimab, one such "super-antibody", is thought to be somewhat robust to neutralization escape [@doi:10.1086/651022] and has been found to be effective against all variants assessed as of August 12, 2021 [@url:https://covdb.stanford.edu/page/susceptibility-data].
```

Another antibody (ADG-2) targets a highly conserved epitope that overlaps the hACE2 binding site of all clade 1 sarbecoviruses [@doi:10.1126/science.abf4830].
Prophylactic administration of ADG-2 in an immunocompetent mouse model of COVID-19 resulted in protection against viral replication in the lungs and respiratory burden.
Since the epitope targeted by ADG-2 represents an Achilles’ heel for clade 1 sarbecoviruses, this antibody, like sotrovimab, might be a promising candidate against all circulating variants as well as emerging SARS-related coronaviruses.

> To date, it has fared well against the Alpha, Beta, Gamma, and Delta variants [@url;https://covdb.stanford.edu/page/susceptibility-data/].

Reviewer 3: ```suggestion
To date, it has fared well against the Alpha, Beta, Gamma, and Delta variants [@url:https://covdb.stanford.edu/page/susceptibility-data].
```

| Casirivimab and imdevimab | N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Reduced viral load at interim analysis |
| Bamlanivimab and etesevimab | B & N/A | Biologic, monoclonal antibodies | EUA | RCT | Supported: Phase 2 clinical trial showed reduction in viral load |
| SNG001 | B | Biologic, interferon | None | RCT | Mixed results: support from initial RCT but no effect found in WHO's Solidarity trial |

> | Sotrovimab | N/A | Biological, monoclonal antibody | EUA | RCT | Supported: Phase 2/3 clinical trial showed reduced hospitalization/death

Author: Strovimab does not seem to be on the website!
